packages/std/src/ibc.rs

```diff
@@ -48,6 +48,18 @@ pub enum IbcMsg {
     /// This will close an existing channel that is owned by this contract.
     /// Port is auto-assigned to the contract's IBC port
     CloseChannel { channel_id: String },
+    /// This writes the ibc acknowledgement for the given packet to the chain.
+    /// This is only needed if you need to delay the acknowledgement.
+    /// In this case, you can return an [`IbcReceiveResponse`] with an acknowledgement of `None`
+    /// from `ibc_packet_receive` and call this message later.
+    WriteAcknowledgement {
+        /// The channel on the destination chain (the chain this is executed on)
+        /// to write the acknowledgement to.
+        dest_channel: IbcEndpoint,
+        /// The sequence number of the packet to acknowledge.
+        sequence: u64,
+        ack: IbcAcknowledgement,
```

Comment:
Here we should use just Binary. If this additional metadata from `IbcAcknowledgement`, this is stuff provided by the chain to the contract and not something the contract developer should need to add

Reply:
Great catch. No idea why I used that type here. Makes no sense.